packages/engine-server/src/DendronEngineV3.ts

```diff
@@ -977,7 +977,9 @@ export class DendronEngineV3 extends EngineV3Base implements DEngine {
         const cachePath = path.join(vpath, CONSTANTS.DENDRON_CACHE_FILE);
         const notesCache = new NotesFileSystemCache({
           cachePath,
-          noCaching: DConfig.readConfigSync(this.wsRoot).noCaching,
+          // noCaching: DConfig.readConfigSync(this.wsRoot).noCaching,
+          // TODO: clean up
+          noCaching: false,
```

Comment:
can we add a next task to get rid of this option? caching should always be on

Reply:
Git it. I left it there to avoid scope creep.